cmd/ingester/app/builder/builder.go

```diff
@@ -64,7 +64,6 @@ func CreateConsumer(logger *zap.Logger, metricsFactory metrics.Factory, spanWrit
 	}
 
 	factoryParams := consumer.ProcessorFactoryParams{
-		Topic:          options.Topic,
```

Comment:
In our environment we don't use the ingester app as is because we create [sarama consumer](https://github.com/jaegertracing/jaeger/blob/main/pkg/kafka/consumer/config.go#L29) in a custom way mainly due to dynamic discovery of Kafka brokers. This change is really about removing the blocker for us to be able to consume from multiple Kafka topics. Let me update the description with this clarity.

Reply:
Updated the issue with more clarity.